src/main/webapp/app/faq/faq.component.ts

```
@@ -0,0 +1,124 @@
+    selector: 'jhi-faq',
+    templateUrl: './faq.component.html',
+    styleUrls: ['./faq.component.scss'],
+    standalone: true,
+    imports: [ArtemisSharedModule, CustomExerciseCategoryBadgeComponent, ArtemisSharedComponentModule, ArtemisMarkdownModule],
+})
+export class FAQComponent implements OnInit, OnDestroy {
+    faqs: FAQ[];
+    filteredFaqs: FAQ[];
+    existingCategories: FAQCategory[];
+    courseId: number;
+    hasCategories: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)